diagram_creation.md

```diff
@@ -0,0 +1,35 @@
+# Creating SVG diagrams
+
+BODS diagrams are created in Scalable Vector Graphic (SVG) format for ease of scaling and translation. SVGs use XML syntax. Text can therefore be stripped out and translated using standard tools. The following workflow guidance is provided so that diagrams within the BODS documentation can accommodate translated text.
```

Comment:
```suggestion
BODS diagrams are created in [Scalable Vector Graphic (SVG) format](https://www.w3.org/TR/SVG2/) for ease of scaling and translation. SVGs use XML syntax. Text can therefore be stripped out and translated using a wide range of text and graphics editors. The following workflow guidance is provided so that diagrams within the BODS documentation can accommodate translated text.
```

Reply:
I edited along these lines.